indra/newview/app_settings/shaders/class1/deferred/pbrterrainF.glsl

```diff
@@ -233,7 +233,7 @@ void main()
 #if (TERRAIN_PBR_DETAIL >= TERRAIN_PBR_DETAIL_NORMAL)
         mix2.vNt = mikktspace(mix2.vNt, vary_tangents[0]);
 #endif
-        mix = mix_pbr(mix, mix2, tm.weight.x);
+        pbr_mix = mix_pbr(mix, mix2, tm.weight.x);
```

Comment:
Hmm, this doesn't look right. Should be `pbr_mix = mix_pbr(pbr_mix, mix2, tm.weight.x);`. `mix` variable is still there in the first argument. Seems to be the case for `tm.weight.y`, etc as well.